### PR TITLE
deprecate docs for GKE and point all links to the new gcloudKey integration

### PR DESCRIPTION
**Pull request by mohit5it9**
https://github.com/Shippable/docs/issues/697

### CODE REVIEW

**Comment by avinci** [comment]
why are we calling this google cloud? why is this on Google Cloud Key?

**Comment by mohit5it9** [comment]
@avinci as users won't be able to create new GKE integrations they can create only gcloudKey integrations now, which will work in exactly the same way as the old gke

**Comment by avinci** [comment]
I am not sure what the convention now is. @rageshkrishna can you let us know what the convention is? we have AWS Keys and now we are calling this google cloud, i am not sure what we called hip chat and slack. 

I want to make sure we are naming this correct. We need a convention that has this done nailed down. whats the internal name and what is the external name conventions.

**Comment by rageshkrishna** [comment]
@avinci The display name of the integration should be the name of the service with which we are integrating. Whether that integration uses a token, key pair, user/password, or key file doesn't need to be part of the name that the user sees. Having this as part of the internal name (amazonKeys, quayLogin, etc.) makes sense internally as we know from the name exactly what to expect in the code.

HipChat and Slack follow this pattern (they are not called HipChat Key or Slack Key externally).

In conclusion, what I'm proposing as the convention is that the internal name be fully self-descriptive, and the external name be the name of the service alone. This means we should rename "AWS Keys" to "AWS".

Let me know what you think.

**Comment by avinci** [comment]
I am ok on this. We need to do this quick. We cannot keep telling our users different logins etc. also JFrog should b jfrog. Not artifactory.
